pkg/update/update.go

```diff
@@ -176,15 +176,20 @@ outer:
 	return &tx, nil
 }
 
-func (c *updater) Apply(ctx context.Context, tx *UpdateTransaction) ([]ctrlpkg.Package, error) {
-	return c.apply(ctx, tx, false)
+func (c *updater) Apply(ctx context.Context, tx *UpdateTransaction, DryRun bool) ([]ctrlpkg.Package, error) {
```

Comment:
Since we are now having two bool options (blocking and dryrun), I suggest the following refactoring: there should only be one `Apply` method instead of two (`Apply`, `ApplyBlocking`), and it gets an `ApplyUpdateOptions` object.

```
type ApplyUpdateOptions struct {
    Blocking bool
    DryRun bool 
}
```

Of course that also means that at any point where `Apply` or `ApplyBlocking` has been called so far, it needs to be changed to use `Apply` with the correct options object.